### PR TITLE
Tud add services

### PR DESCRIPTION
**Pull request by jimiray**
# Description of change

Fetch and Store a TUD Account: 
- services 
- account_uuid 

## Original issue(s)

[Query & add MPI data to TUD account resource](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18967)

### CODE REVIEW

File: modules/test_user_dashboard/spec/services/create_test_user_account_spec.rb [hunk 1]
```diff
@@ -13,9 +13,13 @@
   end
```
Comment (bramleyjl) [inline modules/test_user_dashboard/spec/services/create_test_user_account_spec.rb:13]: The test failure looks like it's happening here, I think we need to change this to be expecting the CreateTestUserAccount service and not a `TudAccount` instance?

**Review by bramleyjl — CHANGES_REQUESTED** [review]

File: modules/test_user_dashboard/app/models/test_user_dashboard/tud_account.rb [hunk 2]
```diff
@@ -9,11 +9,41 @@ class TudAccount < ApplicationRecord
...
+    def user_identity
+      unless (identity = UserIdentity.find(account_uuid))
+        identity = UserIdentity.create(uuid: account_uuid,
+                                       email: email,
+                                       first_name: first_name,
+                                       last_name: last_name,
+                                       gender: gender,
+                                       birth_date: birth_date.to_s(:iso_8601),
+                                       ssn: ssn,
+                                       loa: { lowest: 1, highest: 3 })
```
Comment (ericboehs) [inline modules/test_user_dashboard/app/models/test_user_dashboard/tud_account.rb:30]: As a fellow writer of beautiful code, you can do this and rubocop will be happy:

```suggestion
        identity = UserIdentity.create(
          uuid: account_uuid,
          email: email,
          first_name: first_name,
          last_name: last_name,
          gender: gender,
          birth_date: birth_date.to_s(:iso_8601),
          ssn: ssn,
          loa: { lowest: 1, highest: 3 }
        )
```

**Review by ericboehs — APPROVED** [review]
For the seed, you could amend the Dangerfile to allow db/seeds dirs.

**Review by bramleyjl — APPROVED** [review]
LGTM

**Pull request merged**